lms/djangoapps/courseware/views/views.py

```diff
@@ -1887,10 +1891,18 @@ def _get_fa_header(header):
 
 
 @login_required
-def financial_assistance(_request):
+def financial_assistance(request, course_id=None):
     """Render the initial financial assistance page."""
+    reason = None
+    apply_url = reverse('financial_assistance_form')
+    if course_id and _use_new_financial_assistance_flow(course_id):
+        _, reason = is_eligible_for_financial_aid(course_id)
+        apply_url = reverse('financial_assistance_form_new', args=[course_id])
+
     return render_to_response('financial-assistance/financial-assistance.html', {
-        'header_text': _get_fa_header(FINANCIAL_ASSISTANCE_HEADER)
+        'header_text': _get_fa_header(FINANCIAL_ASSISTANCE_HEADER),
+        'apply_url': apply_url,
+        'reason': reason
     })
```

Comment:
Oh, so you are using the same view for the old and new flow?

Reply:
Yes, just making the minimal changes here and there.